go/vt/proto/vtrpc/vtrpc.pb.go

```diff
@@ -39,7 +39,9 @@ const (
 
 // Code represents canonical error codes. The names, numbers and comments
 // must match the ones defined by grpc (0-16):
-//   https://godoc.org/google.golang.org/grpc/codes.
+//
```

Comment:
you should revert this change (it's unrelated to your PR)

Reply:
this is done by gofmt.

Reply:
i understand that, but the commit hooks _should_ only be running on packages you actually touched

Reply:
which is why [this is failing](https://github.com/vitessio/vitess/actions/runs/3153516810/jobs/5131699790#step:23:1493)